src/drivers/mediatek/mt8195/timer.c

```diff
@@ -44,7 +44,7 @@ int64_t platform_timer_set(struct timer *timer, uint64_t ticks)
 {
 	uint64_t time;
 	uint32_t hitimeout = ticks >> 32;
```

Comment:
Please add the relevant information you added in PR description also here in the commit description:

```
Fix cppcheck bug in mt8195 timer.

Remove redundant initialization for 'lowtimeout'
lowtimeout is overwritten in ' lowtimeout = ticks_set;'
```

Reply:
ok sure.